src/lib/genesis_constants/genesis_constants.ml

```diff
@@ -433,3 +433,19 @@ module Make (Node_config : Node_config_intf.S) : S = struct
 end
 
 module For_unit_tests = Make (Node_config_for_unit_tests)
+
+module Compiled : sig
```

Comment:
This really defeats the point. Please don't do this

Reply:
The plan is to delete this module as soon as we are able to read config from the environment. As I understand it, the previous iteration of `Genesis_constants_compiled` was also only temporary. 

It just didn't seem necessary to split them into different packages, and this change is more of a consequence of how I intentionally broke the code in order to find the places to apply refactoring